src/schema/characters.py

```diff
@@ -20,9 +20,6 @@ class Character(BaseModel):
     inventory_last_updated: Optional[str] = Field(
         None, description="The timestamp of the last update of the inventory"
     )
-    motivation: Optional[str] = Field(
```

Comment:
thought: it would be an interesting expansion to support both overall motivation and adventure goals in the future.

Reply:
thought: totally agree. wearing my "story" hat as opposed to "gamer" hat -- I also wonder if there's some valuable difference between:

- Goal, as in "what the character must do"
- Goal, as in "what the character wants to do"
- Goal, as in "what the story is attempting to make happen"

etc.

In storytelling, the framing is often less about a "goal" and more about:

- A character with some deficits (internal or external)
- A charge for some kind of change that is difficult because of those deficits
- A series of problems that place the character in direct confrontation with the deficits as a required hurdle to create that change